### ISSUE TITLE
Не нашёл двигатель для кисти

### ISSUE THREAD

**Issue by Ochasker** [Rou7e/sm7-opmirs-t1#1]
[formulas.pdf](https://github.com/Rou7e/sm7-opmirs-t1/files/13537163/formulas.pdf)
По кисти ни один двигатель не подошёл. Я проверю, но сомневаюсь, что там правда не подходит ни один.

**Comment by Ochasker** [comment]
[25 Вариант входные данные.pdf](https://github.com/Rou7e/sm7-opmirs-t1/files/13537187/25.pdf)
Входные данные на 25 вариант

**Comment by Ochasker** [comment]
[ДЗ1 Федоров С Вар 25.docx](https://github.com/Rou7e/sm7-opmirs-t1/files/13588563/1.25.docx)
Вручную сделал ДЗ, проблем с выбором двигателей не возникло. Сегодня Филимонов зачёл.

**Comment by Rou7e** [comment]
Благодарю, постараюсь перепилить

**Comment by Rou7e** [comment]
Так, пока это условие не работает, валится на расчете локтя. Видимо все-таки придется реализовать перебор лямбд
